src/oatpp/core/base/Environment.cpp

```diff
@@ -76,24 +106,19 @@ void DefaultLogger::log(v_uint32 priority, const std::string& tag, const std::st
 
   switch (priority) {
     case PRIORITY_V:
-      std::cout << "\033[0m V \033[0m|";
-      break;
-
     case PRIORITY_D:
-      std::cout << "\033[34m D \033[0m|";
-      break;
-
     case PRIORITY_I:
-      std::cout << "\033[32m I \033[0m|";
-      break;
-
     case PRIORITY_W:
-      std::cout << "\033[45m W \033[0m|";
-      break;
-
     case PRIORITY_E:
-      std::cout << "\033[41m E \033[0m|";
-      break;
+#ifdef _WIN32
+      SetConsoleColor(LOG_CONST_TABLE[priority][0]);
+      std::cout << " " << (char)LOG_CONST_TABLE[priority][1] << " ";
+      SetConsoleColor(FOREGROUND_DEFAULT);
+      std::cout << "|";
+#else
+      std::cout << LOG_CONST_TABLE[priority][0];
+      std::cout << " " << LOG_CONST_TABLE[priority][1] << " " << FOREGROUND_DEFAULT << "|";
+#endif
```

Comment:
Looks like there is no `break` statement. 
Also, we can use `if/else` here instead of `switch`

Reply:
Hello, my pr has been open for a long time, can it be merged?